src/nullable_format.rs

```diff
@@ -614,6 +620,77 @@ where
     }
 }
 
+// Implement various traits to allow nullable to validate anything with serde_valid.
+#[cfg(feature = "serdejson")]
+impl<T> Validate for Nullable<T>
+where
+    T: Validate,
+{
+    fn validate(&self) -> Result<(), validation::Errors> {
+        match self {
+            Nullable::Present(val) => val.validate(),
+            Nullable::Null => Ok(()),
```

Comment:
This is presumably on the basis that if null wasn't valid, that should be handled with nullable: false, instead.